packages/venia-ui/lib/components/CartPage/cartPage.css

```diff
@@ -1,13 +1,13 @@
 .root {
-    padding-top: 1.5rem;
-    max-width: 64rem;
+    padding: 2.5rem 3rem;
```

Comment:
I think `3rem` is overkill on mobile. 

![](https://i.gyazo.com/3388d307947d6785f9eb2e45d6820424.png)

Reply:
It is. I see you've set it to `1.5rem`, and that works. It may be able to go as low as `1rem`.